2020/14xxx/CVE-2020-14729.json

```diff
@@ -62,7 +62,7 @@
         "reference_data": [
```

Comment:
MISC or CONFIRM, but name != url for reference.

Reply:
The `name` and `url` fields need to have the same value for `MISC` or `CONFIRM` references.